include/verilated.h

```diff
@@ -1217,6 +1217,29 @@ static inline IData VL_COUNTONES_W(int words, WDataInP lwp) VL_MT_SAFE {
     return r;
 }
 
+// EMIT_RULE: VL_COUNTBITS_II:  oclean = false; lhs clean
+static inline IData VL_COUNTBITS_I(IData lhs, IData ctrl0, IData ctrl1, IData ctrl2, int msb) VL_PURE {
```

Comment:
Usually the width goes first, and is "lbits" to mean lhs-number-of-bits.  Ditto other similar funcs.

```suggestion
static inline IData VL_COUNTBITS_I(int lbits, IData lhs, IData ctrl0, IData ctrl1, IData ctrl2) VL_PURE {
```